v0/destinations/google_adwords_remarketing_lists/util.js

```diff
@@ -114,20 +114,44 @@ const gaAudienceProxyRequest = async request => {
     headers,
     method,
     jobId,
-    body
+    data
   );
   // console.log(JSON.stringify(secondResponse.response.response));
   if (
     !secondResponse.success &&
-    !isHttpStatusSuccess(secondResponse.response.status)
-  )
+    !isHttpStatusSuccess(secondResponse.response.response.status)
+  ) {
     return secondResponse;
+  }
 
   // step3: running the job
   const thirdResponse = await runTheJob(endpoint, headers, method, jobId);
   return thirdResponse;
 };
 
+const gaAudienceProxyRequestHandler = async request => {
+  let response;
+
+  response = await gaAudienceProxyRequest(
+    request,
+    Object.values(request.body.JSON)[0]
+  );
+
+  if (!response.success) {
+    return response;
+  }
+
+  // If two operations are needed i.e create and remove both
+  if (Object.keys(request.body.JSON).length > 1) {
+    response = await gaAudienceProxyRequest(
+      request,
+      Object.values(request.body.JSON)[1]
+    );
+  }
+
+  return response;
```

Comment:
This logic doesn't look right to me. If some user sends both add and remove calls but due to some mistakes in attributes, the add call failed. But the remove call doesn't have any mistakes though we are discarding both the calls. If we can do separate calls for add and remove that would be ideal. In facebook custom audience, we support this. CC @utsabc

Reply:
As discussed we will handle this at transformer and keep things clean at Proxy

Reply:
modified to handle at transformer itself